features/history.json

```diff
@@ -0,0 +1,8 @@
+{
+    "_meta": {
+        "description": "Control for history feature on supported platforms.",
+        "sampleExcludeRecords": {}
+    },
+    "state": "enabled",
```

Comment:
We should probably scope this down to just the platforms where this matters. You can add the feature and `state: enabled` to `ios-override.json`.